README.md

```diff
@@ -24,6 +24,7 @@ A consistent Julia interface for lossless encoding and decoding of bytes in memo
 | BZ2 | .bz2 bzip2 | ChunkCodecLibBzip2 | ✅ | ✅ |
 | Brotli | .br RFC7932 | ChunkCodecLibBrotli | ✅ | ✅ |
 | Blosc |  | ChunkCodecLibBlosc | ✅ | ✅ |
+| Blosc2 |  | ChunkCodecLibBlosc2 | ✅ | ✅ |
```

Comment:
```suggestion
```

This is updated when the package is registered.

Reply:
Done.